python/ray/rllib/rollout.py

```diff
@@ -100,20 +100,19 @@ def run(args, parser):
 
 
 def rollout(agent, env_name, num_steps, out=None, no_render=True):
+
     if hasattr(agent, "local_evaluator"):
         env = agent.local_evaluator.env
+        multiagent = agent.local_evaluator.multiagent
+        if multiagent:
+            policy_agent_mapping = agent.config["multiagent"]["policy_mapping_fn"]
```

Comment:
@ericl this will fail when configuration is reloaded from params.json: mapping function isn't serialized into something directly usable. How can we get mapping reliably here?

Reply:
This looks ok to me -- what do you mean not directly usable?